src/test/java/seedu/fridgefriend/command/HistoryCommandTest.java

```diff
@@ -18,13 +18,8 @@
     private Fridge fridge;
 
     @BeforeEach
-    public void setUp() {
+    public void setUp() throws InvalidDateException, InvalidQuantityException, RepetitiveFoodIdentifierException {
```

Comment:
Do be slightly careful that order matters in the History command, but seems like the checks pass, so looks good!